### PR TITLE
Prod 1787 header style ->PROD-1551_header

### PR DESCRIPTION
**Pull request by vas3a**

### CODE REVIEW

File: src-ts/config/constants.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export enum ToolTitle {
-    settings = 'Account Settings',
+    settings = 'Profile Settings',
```
Comment (testflyjets) [inline src-ts/config/constants.ts:2]: This change should be reverted -- the Figma comp is out of date. This should say "Account Settings".

**Review by testflyjets — CHANGES_REQUESTED** [review]
Need to fix the account settings title and limit the width of the header components.

File: src-ts/lib/styles/_layout.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 @import 'palette';
 
 $header-height: 80px;
+$max-content-width: 1440px;
```
Comment (brooketopcoder) [inline src-ts/lib/styles/_layout.scss:4]: plz use the $xxl-min: 1440px; variable in the _breakpoints.scss file

**Review by brooketopcoder — CHANGES_REQUESTED** [review]
This looks really good! In addition to the suggestions, we also need to constrain the header content to not extend wider than the page content per discussion.

Thanks!

File: src-ts/lib/styles/_layout.scss (hunk 2 above)
Comment (vas3a) [reply src-ts/lib/styles/_layout.scss:4]: @brooketopcoder like $max-content-width: $xxl-min, or should I use directly $xxl-min?

Comment (brooketopcoder) [reply src-ts/lib/styles/_layout.scss:4]: @vas3a i think just use the variable directly--no need for a duplicate

**Review by brooketopcoder — APPROVED** [review]

**Review by brooketopcoder — CHANGES_REQUESTED** [review]
Please style the mobile header per figma. Thanks!

**Review by brooketopcoder — APPROVED** [review]

**Review by testflyjets — APPROVED** [review]

**Pull request merged**